plugins/python_3_script/Dockerfile

```diff
@@ -1,29 +1,20 @@
-FROM rapid7/insightconnect-python-3-38-plugin:5
+FROM rapid7/insightconnect-python-3-38-plugin:4
```

Comment:
why downgrade to version 4?

Reply:
please also set up SDK version in plugin.spec.yaml

Reply:
It got reverted during refresh and I forgot to fix it :( fixed now!